test-configs.yaml

```diff
@@ -1075,6 +1080,11 @@ device_types:
       - blacklist: *allmodconfig_filter
       - blacklist: {defconfig: ['multi_v7_defconfig+CONFIG_SMP=n']}
 
+  stm32mp157c-dk2:
+    mach: stm32
+    class: arm-dtb
+    boot_method: uboot
```

Comment:
Tested OK on staging: https://staging.kernelci.org/test/plan/id/5e981e5d2c87c85ed4b2b40c/
Device type in upstream LAVA.